packages/manager/modules/enterprise-cloud-database/src/service/details/insights/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
If not needed should be removed :)

Reply:
Translations have been added to this file now :)